docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Register a foreign asset
+description: An in-depth guide to registering a foreign asset on the Asset Hub parachain, providing clear, step-by-step instructions.
+---
+
+# Register a foreign asset on Asset Hub
+
+## Introduction
+
+As outlined in the [Asset Hub Overview](./overview.md#foreign-assets){target=\_blank}, Asset Hub supports two categories of assets: local and foreign. Foreign assets originated outside of Asset Hub and are recognized by [Multilocations](https://wiki.polkadot.network/docs/learn/xcm/fundamentals/multilocation-summary){target=\_blank}. This guide will walk you through the process of registering a foreign asset on the Asset Hub parachain.
```

Comment:
I think that we don't use `{target=\_blank}` for links to local pages

Reply:
Mm as far as I understand, we don't use `{target=\_blank}` for links to the same page, as here I'm referring to the page `overview.md` from `register-a-foreign-asset.md`, then I guess it should be included.

Here is the reference: https://github.com/papermoonio/documentation-style-guide/blob/main/style-guide.md#links